### ISSUE TITLE
How to not support html rendering in markdown and just show it as normal text

### ISSUE THREAD

**Issue by gautam-bmdi** [nhn/tui.editor#2799]
Due to backward compatibility with our customers, we want to not render the html in the markdown string but rather simply show the html as text by escaping it.
What would be the best way to configure this behavior on the toast UI editor.
The html inside code blocks is already escaped as per the normal markdown specs. But we want to escape and show the html in the markdown to be escaped and shown as is.

Example, I want the following markdown: 

----

This renders as \<b>bold\</b>.
\```
You can see the html here: \<b>bold\</b>
\```

----

To be rendered as this:

----
This renders as \<b>bold\</b>.
```
You can see the html here: <b>bold</b>
```
----

Instead of:

----
This renders as <b>bold</b>.
```
You can see the html here: <b>bold</b>
```
----

**Comment by gautam-bmdi** [comment]
@jwlee1108 
(Apologies for tagging)

**Comment by gautam-bmdi** [comment]
@js87zz

**Comment by jwlee1108** [comment]
@gautam-bmdi 
If what I understood is correct, I think we can add escape character. (`\`) 


![image](https://user-images.githubusercontent.com/30035674/198279735-57ba3173-1271-49c3-9795-1e2197ca902c.png)

**Comment by gautam-bmdi** [comment]
> 

@jwlee1108 

Yes, but we have historic markdown stored in our backend that has html like `<b>bold<\b>` which we want to escape and show as is in the preview/ wysiwyg and not actually render the word bold as bold. So we do not want to support html in the markdown but rather escape it.

Is there a way to configure that in the editor?
